app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,96 @@
+# TASKS = [
+#     {task: "Clean the dishes"},
+#     {task: "Take out the trash"},
+#     {task: "Take the kids to the park"}
+# ]
+
+class TasksController < ApplicationController
+    def index
+        @tasks= Task.all
+    end
+    
+    def new
+        @task = Task.new
+    end
+
+    def create
+        @task = Task.new(name: params[:task][:name],description: params[:task][:description],completed_at: params[:task][:completed_at]) 
+        if @task.save
+          redirect_to task_path(@task.id)
+          return
+        else 
+          render :new 
```

Comment:
Just a suggestion, include a status code:

```suggestion
          render :new, status: :bad_request
```